CHANGELOG.md

```diff
@@ -2,8 +2,12 @@
 
 ## UNRELEASED
 
-### Fixes
+### Adds
 
+* Modules can now have a `before: "module-name"` property in their configuration to run (initialization) before another module. That works only for target modules that are initializable. 
```

Comment:
"Not initializable" means an "abstract base class" module, right? So the concept isn't relevant for them and it's OK to skip the warning I think.